acceptance/testdata/restorer/container/layers/cacher_buildpack/uncached-layer.toml

```diff
@@ -0,0 +1,3 @@
+build = false
+launch = true
```

Comment:
Why is `launch` `true` here and `false` in `acceptance/testdata/restorer/container/layers/cacher_buildpack/cached-layer.toml`?

Reply:
It was from my test data, I went ahead and made the 2 layers identical except for name and the fact that one exists in the cache being restored and one doesn't. I hope the intent is now more clear :)